src/databricks/labs/remorph/reconcile/recon_capture.py

```diff
@@ -47,10 +45,13 @@ def _read_unmatched_df_from_volumes(
     return spark.read.format("parquet").load(path)
 
 
-def clean_unmatched_df_from_volume(workspace_client: WorkspaceClient, path: str):
+def clean_unmatched_df_from_volume(spark: SparkSession, path: str):
```

Comment:
Make this private

Reply:
Create class for all volumes operations. So changed it .